README.md

```diff
@@ -54,7 +54,16 @@ The easiest way to run Arktos is to bring up a single-node cluster in your local
 
 ```
 cd $GOPATH/src/github.com/arktos
-hack/arktos-up.sh
+./hack/arktos-up.sh
+```
+
+The above command shows how to set up arktos with default network solution, bridge. With release 0.9, an advanced network solution,
+[Mizar](https://github.com/CentaurusInfra/mizar), is introduced into arktos. The integration with Mizar allows tenant pods/services to
+be truely isolated from pods/services in another tenant. To start arktos cluster with Mizar, make sure you are using Ubuntu 18.04+, run the following command:
+
+```
+cd $GOPATH/src/github.com/arktos
+CNIPLUGIN=mizar ./hack/arktos-up.sh
 ```
 
 After the Arktos cluster is up, you can access the cluster with [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) tool just like what you do with a Kubernetes cluster. For example:
```

Comment:
this is actually needed to be changed. in Arktos, there are some changes in Kubectl so we should recommend to use the locally built Kubectl tool, instead of the community version

Reply:
updated